napari_plugin_devtools/__main__.py

```diff
@@ -2,111 +2,231 @@
 napari plugin command line developer tool.
 """
 import argparse
-import json
-import os
+import importlib.util as importlib
 import sys
 
-from .validation import Options, list_hook_implementations, validate_package
+from importlib_metadata import PackageNotFoundError, metadata
+from pkg_resources import DistributionNotFound, get_distribution
+from termcolor import colored
+
+from .validation import Options, list_hook_implementations
+
+PASSED = colored('PASSED', 'green')
```

Comment:
Love the colors! makes it easier to read quickly

Reply:
yes! love this